src/components/DateRangeInput/Presets.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import Box from '../Box';
+
+type OnSelectCallback = (date: [Dayjs, Dayjs]) => void;
+
+interface PresetsProps {
+  currentDateRange: [Dayjs, Dayjs];
+  onSelect: (dates: [Dayjs, Dayjs]) => void;
+}
+
+interface ListItemProps {
+  selected?: boolean;
+  onSelect: (evt: React.KeyboardEvent) => void;
+}
+
+const ListItem: React.FC<ListItemProps> = ({ selected, onSelect, ...rest }) => (
+  <Box
+    as="li"
+    mb={6}
+    cursor="pointer"
+    color="navyblue-100"
+    _selected={{ color: 'gray-50' }}
+    _hover={{ color: 'gray-50' }}
+    aria-selected={selected}
+    onClick={onSelect}
+    fontSize="medium"
+    {...rest}
+  />
+);
+
+const now = dayjs();
+const lastDay = now.subtract(1, 'day');
```

Comment:
Presets as per design needs.

Reply:
You are right, thus I created a helper method that generates the data on the call site.